src/Apps/Order/Components/PaymentPicker.tsx

```diff
@@ -371,7 +371,8 @@ const PaymentPickerWithInnerRef: React.SFC<
   <PaymentPicker ref={innerRef} {...props as any} />
 )
 
-export const PaymentPickerFragmentContainer = createFragmentContainer(
+// FIXME: After deleting `react-tracking.d.ts, this needed to be cast due to HOC
+export const PaymentPickerFragmentContainer: any = createFragmentContainer(
```

Comment:
@ds300 - After removing the local `d.ts` file this was needed. HOC's are tricky and I couldn't figure out how to fix.

Reply:
Yeah doesn't look easy with how the `track` decorator is typed now.

We could fix that upstream, but for now would be good to turn this `any` into something less dangerous. Could do that by adding a cast inline after the `track()(...)` expression:

![image](https://user-images.githubusercontent.com/1242537/57688297-01612a00-7635-11e9-87fe-19f9cbe86e63.png)